docs/development/api/classes/plugin_registry.PluginManager.md

```diff
@@ -88,7 +88,7 @@ Sends a request to install a plugin from the specified ArtifactHub URL.
 **`static`**
 
 **`example`**
-PluginManager.install('pluginID', ' https://artifacthub.io/packages/headlamp/<repo_name>/<plugin_name>');
+PluginManager.install('pluginID', ' <https://artifacthub.io/packages/headlamp/><repo_name>/<plugin_name>');
```

Comment:
This is a generated file. Also, I don't think this is correct because it's a source code example?

Note: `docs/development/api` is generated, so those changes should be made in the source code instead.  However, I know the doc generation is having issues at the moment, so if you change these ones in the source code, please don't regenerate the docs in this PR.

Reply:
Makes sense, I'll go ahead and roll back the changes in `docs/development/api` then